lib/tasks/change_allow_web_publish_to_false.rake

```diff
@@ -4,8 +4,33 @@ namespace :goodcity do
   desc 'change allow_web_publish to false if package.quantity = 0'
   task change_allow_web_publish_to_false: :environment do
     packages = Package.where("quantity = ? and allow_web_publish = ?", 0, true)
+
+    # code to create log for the rake
+    start_time = Time.now
+    File.open("rake_log.txt", "a+"){|f|
```

Comment:
I think it would be better to use the existing ruby `Logger` class for this:

    rake_logger = Logger.new("#{Rails.root}/log/rake_log.log")
    rake_logger.info('log message')